packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -142,12 +142,20 @@ export class DetailsColumnBase extends BaseComponent<IDetailsColumnProps> {
     if (this.props.isDropped) {
       if (this._root!.current!) {
         this._root!.current!.classList!.add(classNames.borderAfterDropping);
+
+        this._async.setTimeout(() => {
+          if (this._root!.current!) {
+            this._root!.current!.classList!.add(classNames.noBorderAfterDropping);
+          }
+        }, 20);
       }
+
       this._async.setTimeout(() => {
         if (this._root!.current!) {
           this._root!.current!.classList!.remove(classNames.borderAfterDropping);
+          this._root!.current!.classList!.remove(classNames.noBorderAfterDropping);
         }
-      }, 1500);
+      }, 1520);
```

Comment:
@Vitalius1 is this the preferred way to trigger CSS3 transitions when using `mergeStyles`? If so, perhaps we should pass the value to the styles function to _at least_ keep them in sync.

Reply:
Is there a reason for changing this to 1520?

Reply:
Since we are adding another border `noBorderAfterDropping` after an interval of 20ms, and the transition duration is 1500ms, hence the minimum safe interval was 1520ms.